### DISCUSSION TITLE
After signin, can visit all rodauth views

### DISCUSSION THREAD

**Discussion by candrawana** [janko/rodauth-rails#149] [Q&A] [answered]
Hi Janko, this is my first time using rodauth-rails.

I have been following the guide, but there is some odds in my rails app:

1. After signin, the account can still get an access to pages like login or create-account
2. When i'm visit via url on rodauth views there a character added (A) in my html, specifically in my footer

![Screenshot (548)](https://user-images.githubusercontent.com/1933232/186968047-c1b462b0-29c6-410b-9bda-190e3c731ef6.png)

````
<footer class="container mx-auto p-5 block border-t">
  <p class="block">
    © 2022 - <%= Time.current.strftime('%Y') %> Gorasa. Website by CDRWN Studio.
  </p>
</footer>
````

But no error detected on my terminal.

```
ruby "3.1.2"
gem "rails", "~> 7.0.3", ">= 7.0.3.1"
gem "propshaft"
gem "rodauth-rails", "~> 1.3"
```

Thanks.

**Comment by janko** [comment, marked as answer]
> After signin, the account can still get an access to pages like login or create-account

By default, Rodauth doesn't prevent access to login and create account pages when the user is logged in. If you want, you can override `already_logged_in` hook to do a redirect, for example:

```rb
already_logged_in do
  set_redirect_error_flash "Already logged in"
  redirect "/"
end
```

> When i'm visit via url on rodauth views there a character added (A) in my html, specifically in my footer

I don't know why is that, it doesn't seem like an issue in rodauth-rails. Can you reproduce it in the official demo app?

**Reply by candrawana** [reply]
Thanks Janko, override the hook answer it.
Sorry for my second question it's not came from rodauth-rails, it seems came from ascii extended character.

Thanks Janko
